src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1982,6 +1973,35 @@ public void validateDeleteRequest(
     return values;
   }
 
+  private static final String snapshotDataTemplate =
+      "SELECT * FROM `<project>.<snapshot>.<table>` LIMIT <count>";
+
+  public List<Map<String, Object>> getSnapshotTable(
+      Snapshot snapshot, String tableName, Integer count) throws InterruptedException {
```

Comment:
This code assumes `count` is non-null so an unboxed type would be more appropriate here.
```suggestion
      Snapshot snapshot, String tableName, int count) throws InterruptedException {
```

Reply:
Missed this one too!